build.js

```diff
@@ -31,15 +41,16 @@ const config = {
     Buffer: 'Buffer'
   },
   plugins: [
-    plugin(stdLibBrowser)
+    plugin(stdLibBrowser),
     /* esbuild-plugin-replace gave broken output
-     * https://github.com/naecoo/esbuild-plugin-replace
+     * https://github.com/naecoo/esbuild-plugin-replace */
     replace({
+      include: /\.js$/, // <-- This little fella here was apparently very important :)
       __ENV__: `${production ? 'production' : 'dev'}`,
       __VERSION__: `${version}`,
       __COMMIT__: `${commit}`,
+      __TEST_CSS__: `${await loadStaticFileData()}`
```

Comment:
Maybe rename `__TEST_CSS__` to  `__SCOPED_CSS__`. It's not test if it works :D 

Whoa. there's an await within template litteral on main thread, i didn't know it was possible... O_o kinda makes sense

Reply:
Since the reading of files takes a couple of milliseconds it needs to be awaited. 

Something to take note of here is that this await statement increases the time it takes to build. So there may come a time when this await statement reduces the app's performance to the point that it defeats its own purpose. But that's a problem for another day 😄